Apps/Admin/Server/appsettings.Development.json

```diff
@@ -39,5 +39,10 @@
     "PhsaV2Admin": {
         "TokenBaseUrl": "https://phsa-ccd-dev.azure-api.net/identity",
         "BaseUrl": "https://phsa-ccd-dev.azure-api.net/healthgatewayadmin"
+    },
+    "ChangeFeed": {
+        "Dependents": {
+            "Enabled": true
+        }
     }
```

Comment:
Shouldn't this be added to appsettings.json instead, and overridden here if needed?

Reply:
Same as above